app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -17,7 +17,7 @@
 %>
 <div class="govuk-document-footer<%= direction_class %>">
   <h2 class="visuallyhidden">Document information</h2>
-  <div class="history-information">
+  <div class="history history-information" id="history">
```

Comment:
@fofr no, probably not, I don't think this class is used in government-frontend. I will remove it. Thanks